snap/info.go

```diff
@@ -173,6 +173,7 @@ type Ref struct {
 	Version     string          `json:"version"`
 	Channel     string          `json:"channel"`
 	Epoch       string          `json:"epoch"`
+	Size        int64           `json:"binary_filesize"`
```

Comment:
I see, notice that the "_" in binary_size does match our style preferring "-", also is not a greate name for the field